src/components/DefinedFileCard/DefinedFileCard.tsx

```diff
@@ -100,29 +124,42 @@ const FileCardMeta: FC<FileCardMetaProps> = ({
 export const DefinedFileCard = (props: DefinedFileCardProps) => {
   const classes = useStyles();
   const { title, onVersionUpload, resourceType, securedFile } = props;
+  const { value: file, canRead, canEdit } = securedFile;
+
+  const { openFilePreview } = useFileActions();
 
-  const { canRead, canEdit } = securedFile;
-  const standardFileActions = getPermittedFileActions(canRead, canEdit);
+  const standardFileActions = useMemo(
```

Comment:
My understanding of useMemo is it's not worth for general operations / jsx rendering. Like in this case react as to store and determine if `canEdit` and `canRead` on every render. It would be faster just to create the list again.

The only place I've really found a use for them is ensuring lists & objects identities don't change. So I see here you've probably got all of these memo-ed because they funnel into main card memo. I think my first point still standards for that as well.
Thoughts?